src/librustc_typeck/diagnostics.rs

```diff
@@ -4095,6 +4095,63 @@ assert_eq!(!Question::No, true);
 ```
 "##,
 
+E0609: r##"
+Attempted to access a non-existent field in a struct was performed.
+
+Erroneous code example:
+
+```compile_fail,E0609
+struct StructWithFields {
+    x: u32,
+}
+
+let s = StructWithFields { x: 0 };
+println!("{}", s.foo); // error: no field `foo` on type `StructWithFields`
+```
+
+To fix this error, check if you didn't misspell the field's name or that the
+field actually exist. Example:
+
+```
+struct StructWithFields {
+    x: u32,
+}
+
+let s = StructWithFields { x: 0 };
+println!("{}", s.x); // ok!
+```
+"##,
+
+E0610: r##"
+Attempted to access a field on a primitive type.
+
+Erroneous code example:
+
+```compile_fail,E0610
+let x: u32 = 0;
+println!("{}", x.foo); // error: `{interger}` is a primitive type, therefore
+                       //        doesn't have fields
+```
+
+Primitive types are the most basic types available in Rust and don't have
+fields. If you want to use fields, take a look to the structs. Example:
+
+```
+// We declare struct called `Foo` containing two fields:
+struct Foo {
+    x: u32,
+    y: i64,
+}
+
+// We create an instance of this struct:
+let variable = Foo { x: 0, y: -12 };
+// And we can now call its fields:
+println!("x: {}, y: {}", variable.x, variable.y);
+```
+
+For more information, go read The Rust Book: https://doc.rust-lang.org/book/
```

Comment:
"go read" sounds like a derogatory command. I'd suggest "see" instead (and drop the comma).